packages/client/e2e/credit.spec.ts

```diff
@@ -49,7 +50,44 @@ test.describe("Credit", () => {
   })
 
   test.describe("New RFQ", () => {
+    test.setTimeout(TestTimeout.EXTENDED)
+
     test("Create RFQ for GOOGL @smoke", async () => {
+      const MAX_ATTEMPT = 2
+
+      const clickAcceptButton = async () => {
+        const firstQuote = rfqsPage.getByTestId("quotes").locator("div").first()
+        const acceptButton = firstQuote.getByText(/Accept/)
+
+        await firstQuote.hover()
+        // Wait for first quote response
+        await expect(firstQuote).not.toContainText("Awaiting response", {
+          timeout: ElementTimeout.LONG,
+        })
+        await acceptButton.click({ timeout: ElementTimeout.AGGRESSIVE })
+      }
+
+      // retry logic to circumvent intermittent failures at clicking on the accept button
+      const acceptQuote = async () => {
+        let attempt = 0
+        let isQuoteAccepted = false
+        while (!isQuoteAccepted) {
+          // fail test if max attempt threshold is reach
+          expect(
+            attempt,
+            "Check if max attempts to click on accept button is reach",
```

Comment:
this is the message to [print on error](https://playwright.dev/docs/test-assertions#custom-expect-message), right?
so let's go with something like "Max accept attempts exceeded" or "Exceeded maximum attempts to click on quote Accept button" which is probably better

Reply:
you are right, I've updated the message accordingly